source/_patterns/03-templates/00-layouts/html.html.pl.twig

```diff
@@ -28,7 +28,7 @@
 <html{{ html_attributes }}>
   <head>
     <head-placeholder token="{{ placeholder_token }}">
-    <title>eCMS</title>
+    <title>{{ head_title }}</title>
     {# <title>{{ head_title }}</title> This doesnt work #}
```

Comment:
Remove this line please. Ah yes, this is a carryover from when we had issues with templates from PL being pulling into drupal theme.